### ISSUE TITLE
timespace.R - crash on line 77 

### ISSUE THREAD

**Issue by jaromf** [TaddyLab/BDS#9]
When running line 77 `summary(ARdj <- glm(dja[2:n] ~  d` of timespace.R, R Studio crashes and reports a "Segmentation fault" in the Bash terminal.  I read on StackOverflow that this could be due to limited memory.  I'm running Crostini / Debian on a Pixelbook with 8 GB ram and approx. 80 GB free disk space.  Can anyone suggest a possible solution or troubleshooting tips?

**Comment by jaromf** [comment]
Here's the result of running 'gc(ARdj)`
> gc(ARdj)
Garbage collection 15 = 12+0+3 (level 2) ... 
26.9 Mbytes of cons cells used (53%)
7.6 Mbytes of vectors used (12%)
         used (Mb) gc trigger (Mb) max used (Mb)
Ncells 502063 26.9     944981 50.5   944981 50.5
Vcells 988437  7.6    8388608 64.0  1762250 13.5

**Comment by mataddy** [comment]
hi @jaromf I can't replicate this error and it does indeed look like a memory issue.  The memory that matters is the 8GB -- your working memory.  Nothing in this file is big enough to fill that up so it must be that either (a) you have a bunch of other stuff in R using up working memory or (b) other applications are taking all the working memory.   I'd suspect (a), so quit R studio and when you re-launch make sure that it is not loading any saved objects from previous sessions (i.e., make sure to not save your session).

**Comment by jaromf** [comment]
Hi @mataddy I tried clearing as much RAM as possible, but still run into the issue.  I posted the topic on a related thread in Rstudio Community.  Ultimately, I may need to use a computer with more RAM. I'm thinking about setting up a powerful desktop computer and accessing it remotely as needed. https://community.rstudio.com/t/rstudio-desktop-on-a-chromebook-without-crouton-using-new-linux-beta-option/17393/9
